### DISCUSSION TITLE
Login error with http response 200

### DISCUSSION THREAD

**Discussion by aferrandi** [NightscoutFoundation/xDrip#3068] [Q&A]
I was able to receive the data from the carelink server for two days, then I startet to get the http error 504 and now for two weeks I've got this:

![Screenshot_2023-09-05-17-41-57-31_2dd2c1bcb8f659cd696bcba2072f751b](https://github.com/NightscoutFoundation/xDrip/assets/1436546/77d56f52-1910-4103-8ec7-845b4a826a6a)

I use always the last nightly build.

It looks like it is able to login but after that it can't retrieve the session data. (the REST call"patient/user/me").
What can it be? I configured user, password and country. Do I need to configure also the patient name?

**Comment by Navid200** [comment]
These instructions may be helpful in trying to find out what the cause is:
https://github.com/NightscoutFoundation/xDrip/discussions/2916#discussioncomment-6321592

**Reply by aferrandi** [reply]
I added the debug options suggested in the post but they don't give more information, it seems:

![Screenshot_2023-09-06-16-46-45-34_2dd2c1bcb8f659cd696bcba2072f751b](https://github.com/NightscoutFoundation/xDrip/assets/1436546/6eeaeb47-aa61-43b3-bd2a-7dcf38c2a9b9)

**Reply by Navid200** [reply]
@benceszasz

**Comment by aferrandi** [comment]
I just wanted to add that I have tested it with:

- a different country (USA, Germany...). Same log
- a wrong password. It gets an error in an earlier phase, when it tries to login.

**Comment by NoobDevDan** [comment]
Still no fix or work around for this?
